BlazarUI/app/scripts/components/shared/SearchFilter.jsx

```diff
@@ -40,14 +42,23 @@ class SearchFilter extends Component {
     this.refs.searchFilterInput.focus();
   }
 
-  handleChange() {    
+  handleChange() {
     this.setState({
       searchValue: this.refs.searchFilterInput.value
     });
     this.handleSearchDebounced();
   }
 
-  render() {    
+  handleKeyup(e) {
+    const notFocusedOnInputElement = e.target === document.body || e.target.tagName === 'A';
+    const modifierKey = e.metaKey || e.shiftKey || event.ctrlKey;
+    const shortcutPressed = FOCUS_SEARCH_BAR_SHORTCUTS.includes(e.which);
```

Comment:
Is Array.prototype.includes allowed? It's not supported in IE and babel has a polyfill for it
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility

I can switch it to http://underscorejs.org/#contains if that's safer.